src/Resources/views/CRUD/list_outer_rows_mosaic.html.twig

```diff
@@ -30,7 +30,9 @@ This template can be customized to match your needs. You should only extends the
 
                             <div class="mosaic-inner-box-default">
                                 {% block sonata_mosaic_background %}
-                                    <img src="{{ meta.image }}" alt="" />
+                                    <img src="{{ meta.isImageAvailable ?
```

Comment:
I'm just wondering. We use the `Meta` class outside of block objects? Or what is the type of the `meta` variable?

Reply:
> We use the `Meta` class outside of block objects?

Yes, I think we do.

Reply:
In that case, we should introduce a new `Meta` class for the admin stuff

Reply:
Seems a good idea, in this case I guess no need to change `Metadata` classes from core and block bundles, but I have no idea how to proceed...

Also, I am even not sure about the fact that we use the `Meta` class outside of block objects. How coud I check?